modules/dbfs/couchbase.go

```diff
@@ -77,7 +77,8 @@ func (di *DatabaseImpl) cbInsertNewFile(file cbFile) error {
 		return err
 	}
 
-	_, err = cb.bucket.Insert(strconv.FormatInt(file.FileID, 10), file, 0)
+	// TODO (shapiro): change me back to bucket.Insert(...)
+	_, err = cb.bucket.Upsert(strconv.FormatInt(file.FileID, 10), file, 0)
```

Comment:
this goes back to my comment on it being temporary. when it throws an error, there's no way to resolve that, and the client wont know why it failed other than that it just won't successfully write. 
The only way that this code is run is after the mysql create file call, which uses the autoincrement. So if we're overwriting something, it's because mysql doesn't know about it, meaning that none of the clients know about it either. So there's no real need to worry about overwriting something important because once we have important data on the server this will get replaced back.